### PR TITLE
Add notes for environment variable interpolation and multi-package

### PR DESCRIPTION
**Pull request by samuel-williams-da**

### CODE REVIEW

File: docs/2.9.3/docs/tools/assistant.rst [hunk 1]
```diff
@@ -178,3 +180,51 @@ warning.  See
 `the Daml forum <https://discuss.daml.com/t/making-the-most-out-of-daml-compiler-warnings/739>`__
 for a discussion of the meaning of these warnings and pointers to other
 available warnings.
+
+Multi-Package Config File (``multi-package.yaml``)
+==================================================
+See :ref:`Multi-Package Build<multi-package-build>` for more information on this file.
+The ``multi-package.yaml`` file is used to inform Daml Build and the IDE of projects containing multiple
+connected daml packages.
```
Comment (carrielaben-da) [inline docs/2.9.3/docs/tools/assistant.rst:188]:
```suggestion
connected Daml packages.
```

File: docs/2.9.3/docs/tools/assistant.rst [hunk 2]
```diff
@@ -178,3 +180,51 @@ warning.  See
...
+    - ./path/to/package/a
+    - ./path/to/package/b
+  projects:
+    - ./path/to/project/a
+    - ./path/to/project/b
+
+Here is what each field means:
+
+- ``packages``: an optional list of directories containing Daml packages, and by extension, ``daml.yaml`` config files. These allow Daml Multi-Build to
+  find the source code for dependency dars and build them in topological order.
```
Comment (carrielaben-da) [inline docs/2.9.3/docs/tools/assistant.rst:204]:
```suggestion
  find the source code for dependency DARs and build them in topological order.
```

File: docs/2.9.3/docs/tools/assistant.rst [hunk 3]
```diff
@@ -178,3 +180,51 @@ warning.  See
...
+- ``projects``: an optional list of directories containing ``multi-package.yaml`` config files, which will extend the above package set for resolving
+  the build order. These exist to allow separation of your project into sub-projects, which can still be built when removed from the surrounding environment.
```
Comment (carrielaben-da) [inline docs/2.9.3/docs/tools/assistant.rst:206]:
```suggestion
  the build order. These exist to allow separation of your project into sub-projects which can still be built when removed from the surrounding environment.
```

File: docs/2.9.3/docs/tools/assistant.rst [hunk 4]
```diff
@@ -178,3 +180,51 @@ warning.  See
...
+
+The ``multi-package.yaml`` file supports `Environment Variable Interpolation <#environment-variable-interpolation>`__.
+
+Environment Variable Interpolation
+==================================
+Both the ``daml.yaml`` and ``multi-package.yaml`` config files support environment variable interpolation on all string fields.
+Interpolation takes the form of ``${MY_ENVIRONMENT_VARIABLE}``, which will be replaced with the content of ``MY_ENVIRONMENT_VARIABLE`` from the
+calling shell.
```
Comment (carrielaben-da) [inline docs/2.9.3/docs/tools/assistant.rst:214]:
```suggestion
Interpolation takes the form of ``${MY_ENVIRONMENT_VARIABLE}`` which are replaced with the content of ``MY_ENVIRONMENT_VARIABLE`` from the
calling shell.
```

File: docs/2.9.3/docs/tools/assistant.rst [hunk 5]
```diff
@@ -178,3 +180,51 @@ warning.  See
...
+A simple example is given below:
+
+.. code-block:: yaml
+
+  sdk-version: ${SDK_VERSION}
+  name: ${PROJECT_NAME}_test
+  source: daml
+  version: 1.0.0
+
+Interpolation strings can be escaped with a preceding ``\``, and interpolation can be disabled for a config file
```
Comment (carrielaben-da) [inline docs/2.9.3/docs/tools/assistant.rst:224]:
```suggestion
Interpolation strings can be escaped with a preceding ``\``, and interpolation can be disallowed for a config file
```

**Review by carrielaben-da — APPROVED** [review]
A few small changes

**Comment by samuel-williams-da** [comment]
Thanks! Could you confirm whether this needs to be duplicated to 2.10/3.1? I assume yes

**Comment by carrielaben-da** [comment]
> Thanks! Could you confirm whether this needs to be duplicated to 2.10/3.1? I assume yes

Yes, that's correct.

**Pull request merged**